site/server/devServer.tsx

```diff
@@ -98,6 +99,12 @@ devServer.get("/explore", async (req, res) => {
     res.send(await renderExplorePage())
 })
 
+devServer.get("/explore/indicators.json", async (req, res) => {
+    // TODO remove this doing-undoing madness, it's only here to send correct
+    // mimetype so that the browser extension for syntax highlighting kicks in.
+    res.json(JSON.parse(await renderExplorableIndicatorsJson()))
```

Comment:
I think you can just do `res.type('json').send(…)`

Alternately: `res.format({json: () => …})`

Reply:
Ah, thanks!